chapter10/voice-werewolf/werewolf/game.py

```
@@ -145,15 +164,15 @@ def assign_identities(self):
         wolves = self.wolves()
         team = "、".join(w.name for w in wolves)
         self.wolves_send("狼人队友身份", f"狼人阵营的玩家是：{team}（你们互为队友，夜晚共同行动）")
-        human = next((p for p in self.players if getattr(p, "is_human", False)), None)
-        if human:
+        user = next((p for p in self.players if getattr(p, "is_user", False)), None)
+        if user:
             # The old all-AI observer table would be a side-channel leak to a real
             # player. In live mode only the human's permitted private facts are spoken.
             print("  [真人局] 上帝视角身份表已隐藏，防止终端侧信道泄露。")
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)